Makefile

```diff
@@ -12,6 +12,7 @@ else
 endif
 
 ALPINE_VER ?= 3.18
+GO_ALPINE_VER ?= 3.18
```

Comment:
Looks like this variable is unused.

Reply:
I see it is used in [Makefile](https://github.com/trustbloc/wallet-sdk/blob/main/Makefile#L82) so decided to add.